.circleci/config.yml

```diff
@@ -49,8 +55,19 @@ jobs:
           POSTGRES_USER: administrate
           POSTGRES_DB: ruby24
           POSTGRES_PASSWORD: ""
+    steps:
+      - shared_steps
+      # Run the tests against multiple versions of Rails
+      - run: BUNDLE_GEMFILE=gemfiles/rails42.gemfile bundle install
+      - run: BUNDLE_GEMFILE=gemfiles/rails50.gemfile bundle install
+      - run: BUNDLE_GEMFILE=gemfiles/rails51.gemfile bundle install
+      - run: BUNDLE_GEMFILE=gemfiles/rails52.gemfile bundle install
```

Comment:
Are we not able to use `bundle exec appraisal install` here, like we are with the others?

Reply:
‘bundle exec appraisal install’ fails when trying to install Rails 6 on Ruby 2.4

Reply:
Hmm. I guess this is a bug in `appraisal`.

Reply:
Maybe we could add something like `appraisal install rails42 rails50 rails51 rails52` to install a subset of the appraisals. We have something like that for `appraisal update`.